.github/workflows/test.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         - python-version: '3.12'
         - python-version: '3.8'
           toxenv: min
+        - python-version: '3.8'
+          toxenv: extra
```

Comment:
What do you think about running "extra" on 3.12? 
For example, a new Scrapy could be released which drops support for 3.8; in this case "extra" might have to install older Scrapy, and we'll never check on the latest Scrapy.

Reply:
yes, makes sense, I've added this as additional requirement.